files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -534,28 +534,34 @@ audit-secureblue:
         for f in ${!flatpaks[@]}; do
             has_network=false
             has_x11=false
+            has_hardened_malloc=true
+            status=$STATUS_SUCCESS
             fullref=${flatpaks["$f"]}
             permissions=$(flatpak info --show-permissions "$fullref")
             if hasPermission "$permissions" "shared" "network"; then
                 has_network=true
+                [[ ! "$status" != "$STATUS_FAILURE" ]] && status="$STATUS_WARNING"
             fi
-            if hasPermission "$permissions" "sockets" "x11" && ! hasPermission "$permissions" "sockets" "fallback-x11"  ]]; then
+            if hasPermission "$permissions" "sockets" "x11" && ! hasPermission "$permissions" "sockets" "fallback-x11"; then
```

Comment:
can you put the changes that aren't related to hardened_malloc flatpak checks in another PR to make this easier to review?